definitions/consented/json/CaseEvent.json

```diff
@@ -675,5 +675,89 @@
     "PostConditionState": "*",
     "SecurityClassification": "Public",
     "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "ID": "FR_awaitingInfo",
+    "Name": "Awaiting Information",
+    "Description": "Awaiting Information",
+    "DisplayOrder": 38,
+    "PreConditionState(s)": "applicationSubmitted;applicationIssued;responseReceived;orderMade;consentOrderApproved",
+    "PostConditionState": "awaitingInfo",
+    "SecurityClassification": "Public",
+    "ShowEventNotes": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "ID": "FR_caseSubmissionFromAwaitingInfo",
```

Comment:
Is this supposed to do the same thing as the current event "case submission" (i.e. id: FR_applicationPaymentSubmission), because that one has callbacks. If it is to do the same thing, then you could just add 'awaitingInfo' as a preconditionState to that existing event. 👍

Reply:
Good question, but this Case Submission event (and also all other new events in this PR besides Awaiting Info) is just for moving the case back to the same state it has already been in before Awaiting Information, meaning all necessary files will have already been generated and all actions done. The new Case Submission has no actions but only the state change. Due to that, the previous event couldn't be reused.